packages/core/admin/package.json

```diff
@@ -107,19 +106,19 @@
     "p-map": "4.0.0",
     "passport-local": "1.0.0",
     "pluralize": "8.0.0",
-    "prop-types": "^15.7.2",
+    "prop-types": "^15.8.1",
     "qs": "6.11.1",
-    "react": "^17.0.2",
+    "react": "^18.2.0",
```

Comment:
I don't think we have a defined pattern, with peer-deps typically you want to match the lowest version you support (wider support range that way) – dev deps IMO it doesn't matter _too much_ (some may disagree) and prod deps I think some of them are locked because we can't bump higher – IMHO if we want to take stock and sort this it should be a separate PR 👍🏼